Resources/Prototypes/_NF/PointsOfInterest/bahamamamas.yml

```diff
@@ -10,19 +10,14 @@
 # I want to lie, shipwrecked and comatose, drinking fresh mango juice.
 - type: pointOfInterest
   id: Bahama
+  parent: BasePOI
   name: "Bahama Mama's"
   minimumDistance: 1200
   maximumDistance: 2900
   spawnGamePreset: [ NFAdventure, NFPirate ]
   spawnGroup: RestStop
   gridPath: /Maps/_NF/POI/bahama.yml
   addComponents:
-  - type: IFF
-    color: "#ffa600"
-    readOnly: true
-  - type: Shuttle
-    angularDamping: 999999
-    linearDamping: 999999
   - type: StationTransit
 
```

Comment:
Stuff that have a addcomp list don't get overwrite?

Reply:
It should work like the EntityPrototype, and your ComponentRegistry _does_ accept a list of items.  Should test this better, but I think we're good.

Reply:
Tested a little bit more, works fine.  LPBravo's hidden, BaseShuttleAntag's fine, all antag ships are hidden with white IFF, and the base ones are all visible with white IFF.